src/main/resources/schema/provider.definition.schema.v1.json

```diff
@@ -19,6 +19,13 @@
                         "type": "string"
                     },
                     "additionalItems": false
+                },
+                "timeoutInMinutes": {
+                    "description": "Defines the timeout for the entire operation to be interpreted by the invoker of the handler.  The default is 120 (2 hours).",
+                    "type": "integer",
+                    "minimum": 2,
+                    "maximum": 4320,
```

Comment:
We can't support 72hours with Role based auth. 12h is the maximum supported by IAM/STS.

Reply:
think we also need to be clear which operations are subject to this timeout (CUD, not RL), right?

Reply:
that's true it would not apply to read and list... I'll have to split it into 2 handler definitions then

Reply:
> We can't support 72hours with Role based auth. 12h is the maximum supported by IAM/STS.

we have to support 72 hours at some point to maintain parity with resources we currently support, but I can lower it to 12 hours for the time being